resources/lang/en-US/admin/accessories/table.php

```diff
@@ -1,11 +1,10 @@
 <?php
 
 return array(
+    'accessory_checkout_id'     => 'Checkout ID',
```

Comment:
Could we maybe just use the generic `ID` translation here?

Reply:
Updated:

![update accessories table](https://github.com/user-attachments/assets/b86eb13a-77fb-49c1-bc77-227842e1710a)